.github/workflows/.ci_cd.yml

```diff
@@ -0,0 +1,96 @@
+# Copyright 2017, Backblaze Inc. All Rights Reserved.
+# License https://www.backblaze.com/using_b2_code.html
+
+name: b2-sdk-java ci/cd
+
+on:
+  push:
+  pull_request:
+    branches:
+    - master
+
+env:
+  PYTHON_DEFAULT_VERSION: 3.6
+  OUTPUT_DIR: $GITHUB_WORKSPACE/build/outputs
+  OUTPUT_ZIP: b2-sdk-build-${GITHUB_RUN_NUMBER}.zip
+  # This token was generated under rhryckewicz account
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Now that you've tested out that this token is needed, lets get one from a service account. Can also be done in a follow up PR, well removing the comment would be a follow up PR :)